Add Innr RB 250 C, Hue White and Color Ambiance E27 1600lm, and WiZ RGBW Tunable Globe G95 E27

ahartmanns opened the pull request:
I measured three types of lights with a Shelly Plug S:
* Innr RB 250 C: a Zigbee E14 colour candle light; I measured 3 of them without a dummy load, via Home Assistant with Zigbee2MQTT.
* Hue LCA008: the new E27 1600lm colour bulb; I measured 1 of them with a dummy load (dumb light bulb) to get non-zero standby power, via Home Assistant with Zigbee2MQTT.
* WiZ SHRGB1C: a WiZ wifi colour G95 globe bulb for E27; I measured 1 of them with the same dummy load, via Home Assistant with the native WiZ integration. This appears to be the first WiZ bulb in the library; I hope I have integrated it correctly. I am not sure if the WiZ model names (here: SHRGB1C) are unique per bulb type, or if they are about the internal technology - I guess we have to see as more WiZ bulbs come in. The two bulbs I have (this one and a high-lumen E27 in normal shape) at least have different model names.

bramstroker commented:
Hi thanks a lot for your contribution.

Could you add information about the measure method methodology (number of lights, dummy load) to `measure_description` in the model.json? That would be nice for future reference.

Regarding the model/manufacturer. You can check this in the device screen (under Settings -> Devices & Services -> Devices). Then look at manufacturer and model.

bramstroker commented:
And btw you don't have to edit supported-models.md manually next time, this will be done after I merge the PR automatically by a github action.

dsolva commented:
Thanks a lot for adding Wiz, i have many of this brand and I'm looking to add my 3 different models. In Mexico they are very cheap and got most of them at around 6-7dlls.  I have a "SHRGB1C" according to home assistant and it’s not a G95 Globe, it’s a A19 E27 with regular shape with 800lm. So, as you mention, maybe that name is not unique to bulb type.

I thought that maybe the only difference was the bulb type, but there is a huge difference in idle power, for your model I see almost a Watt while mine measures 0.3W. I still don’t have an extremely precise measurement plug (i will get one soon) but i have tried 3 different ones (smart and non smart) and stacked my 5 bulbs to estimate individual idle consumption, I’m pretty sure its not above .5W.

For idle consumption i would think the ESP packed inside may vary (ESP01 31 according to home assistant, 21098-espwroomm02d to packaging) The esp02d makes a lot of sense since average current is 0.8A and 2.4V-3.6V (0.22V-0.29V).

I don’t know if other esps with less power efficiency, or if for any reason there is a 5.0gz version, or if 220v vs 120v could make a difference.

Also, my model states up to 8.8W while i have measured around 8.2W max.

Here is the link to my model: https://www.wizconnected.com/es-mx/p/luz-led-foco-a19-e26/046677556136

dsolva commented:
> * WiZ wifi colour G95 globe bulb for E27

the "model" number appears in the box. Mine is:
https://www.energystar.gov/productfinder/product/certified-light-bulbs/details/2350177

I think yours is 929002383902 and in amazon states 1100lm and 220V. 

Researching more on the way wiz names the products it come down to regular shape and socket. The Bulb A19 E27 (mine) for  US/Mexico/etc has the equivalent Bulb A60 E27 but a 006 lumen difference. It seems that the second one (220V) according to a random post I saw has 0.4W which is a lot more similar to my 0.3 reading. Almost the same but they would have different consumption.

I would have suggested to group by the simple name by Wiz: "Bulb A19 E26 Color" for mine and yours "Globe G95 E27 Color". However I'm looking at a Bulb A60 E27 for 3 different countries: Brasil (120V-8.8W 800lm like mine but different socket), Peru (220V-8W 806lm) and Spain (220V-9W 806lm). Im not sure if spains 9W in webpage is arbitrary and Peru=Spain just as Brasil=Mexico/US. In that case "Bulb 8.8W-120V Color" (there are 13.5W available) "Globe 11W 220V Color" would be a second option with more posibility of extrapolating by ignoring socket.  Other models that exist would be "Filament 3.8W 120V" or "Reflector 7.5W 120V", "Candle 3.9W 120V".

Hope this helps categorizing them. I want to add my bulb and filament versions.

ahartmanns commented:
I just updated the commits to add the measurement details to `measure_description` and remove the changes in supported-models.md.

Seeing the info by @dsolva, I also changed the WiZ bulb's `name` to "WiZ Globe G95 E27 Color (9290023839)". That number is the one I have on the physical box that the bulb came in. In Home Assistant, the data shown under the "Device info" heading is 
```
SHRGB1C
by WiZ

Firmware: 1.27.0
Hardware: ESP03 01
```
So indeed I fear that the WiZ integration (and maybe the bulbs themselves) do not provide enough information via the model name to uniquely indentify a bulb type; at least "SHRGB1C" is not sufficient. Can powercalc maybe also include other information like the hardware as shown above? Still, it may be useful to get this in to just get the measurement and integration of WiZ bulbs started.

By the way, I just put the WiZ bulb alone behind the Shelly Plug S, and get 0.9 to 1.0 W of power consumption shown in Home Assistant (which appears to round to 1 decimal digit), so the measured 0.96 W idle power appear realistic (if sadly high).

bramstroker commented:
> Can powercalc maybe also include other information like the hardware as shown above? Still, it may be useful to get this in to just get the measurement and integration of WiZ bulbs started.

This is not yet possible and don't think I will add support for that somehow. Also it would not bring any added value imo as the hardware seems just the microcontroller (board) the WiZ light is running on, this won't be unique for each individual light anyhow.
So the only option is to let the users configure this light manual with the yaml configuration and defining the manufacturer and model.

You may also change the directory name from 9290023839 to SHRGB1C as this allows to add the other non-unique SHRGB1C lights to the library.

bramstroker commented:
Perfect thanks
